README.md

```diff
@@ -1,6 +1,6 @@
-# Prettier Chrome Extension
+# Prettier Browser Extension
 
-> Extension that adds support for prettifying input boxes that support Markdown or Code.
+> Extension that adds support for prettifying input boxes that support Markdown or Code. Currently supports Chrome and Firefox.
```

Comment:
I'm 👍 on all the other changes here, but I think we should add a section about the development/deployment status of the Firefox version if we're going to mention it. Alternatively we can revert this line until the Firefox version is published and linked in the readme.